pages/common/http.md

```diff
@@ -1,32 +1,36 @@
 # http
 
-> HTTPie: HTTP client, aims to be easier to use than cURL.
-> More information: <https://httpie.org>.
+> HTTPie: designed for testing, debugging, and generally interacting with APIs & HTTP servers.
+> More information: <https://httpie.io/docs/cli/usage>.
 
-- Download a URL to a file:
+- Make a simple GET request (shows response header and content):
 
-`http --download {{example.org}}`
+`http {{https://example.org}}`
 
-- Send form-encoded data:
+- Print specific output content (request [H]eaders, request [B]ody, response [h]eaders, response [b]ody, response [m]etadata):
 
-`http --form {{example.org}} {{name='bob'}} {{profile_picture@'bob.png'}}`
+`http --print {{H|B|h|b|m|Hh|Hhb|...}} {{https://example.com}}`
 
-- Send JSON object:
+- Specify HTTP method when sending a request:
 
-`http {{example.org}} {{name='bob'}}`
+`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} {{https://example.com}}`
 
-- Specify an HTTP method:
+- Specify additional headers in a request:
 
-`http {{HEAD}} {{example.org}}`
+`http {{https://example.com}} {{'User-Agent: Mozilla/5.0' 'Accept-Encoding: gzip'}}`
 
-- Include an extra header:
+- Authenticate to a server using different authentication methods:
 
-`http {{example.org}} {{X-MyHeader:123}}`
+`http --auth {{username:password|token}} --auth-type {{basic|digest|bearer}} {{GET|POST|...}} {{https://example.com/auth}}`
 
-- Pass a username and password for server authentication:
+- Construct a request but do not send it (similar to a dry-run):
 
-`http --auth {{username:password}} {{example.org}}`
+`http --offline {{GET|DELETE|...}} {{https://example.com}}`
 
-- Specify raw request body via `stdin`:
+- Use named sessions for persistent requests between requests:
 
-`cat {{data.txt}} | http PUT {{example.org}}`
+`http --session={{path/to/logged-in-session.json}} {{--auth username:password https://example.com/auth API-KEY:xxx}}`
```

Comment:
```suggestion
`http --session {{session_name|path/to/session.json}} --auth {{username}}:{{password}} https://example.com/auth {{API-KEY:xxx}}`
```

Reply:
I would like to stick to the original example. Using session does not necessarily require use the of `--auth`. An unauthenticated session can also have relevant session information. Hence I enclosed the whole `{{--auth username:password https://example.com/auth API-KEY:xxx}}` in double braces.

Reply:
Ok, what about this? `--session {{name}}` stores the session file in the default location. 
```suggestion
`http --session {{session_name|path/to/session.json}} {{--auth username:password https://example.com/auth API-KEY:xxx}}`
```